### PR TITLE
Volume-related graph refresh fixes & VCR specs

### PR DESCRIPTION
**Pull request by cben**
Added refresher specs for CVs, PVs, PVCs.
Testing those in the new easier-to-reproduce VCRs.  I was lazy and avoided re-recording but relying on some hawkular/cassandra stuff that's not in the template — but should be pretty stable.

@miq-bot add-label bug, test

Graph refresh RFE: https://bugzilla.redhat.com/show_bug.cgi?id=1470021

### CODE REVIEW

**Comment by cben** [comment]
Kicking Travis as master is now green

**Comment by cben** [comment]
@zeari @enoodle @moolitayer Please review.

**Comment by moolitayer** [comment]
> I was lazy and avoided re-recording but relying on some hawkular/cassandra stuff that's not in the template — but should be pretty stable.

I'm all for setting out base test cluster for infra stuff + template - that's what we deploy mostly anyway.

File: app/models/manageiq/providers/kubernetes/container_manager/inventory_collections.rb [hunk 1]
```diff
@@ -313,11 +313,14 @@ def initialize_inventory_collections
...
-          :parent         => manager,
-          :builder_params => {:ems_id => manager.id},
-          :association    => :persistent_volume_claims,
-          :use_ar_object  => true # serialized :capacity attr
+          :model_class          => PersistentVolumeClaim,
+          :parent               => manager,
+          :builder_params       => {:ems_id => manager.id},
+          :association          => :persistent_volume_claims,
+          :use_ar_object        => true, # serialized :capacity attr
+          # TODO: add container_project_id column to persistent_volume_claims.
```
Comment (moolitayer) [inline app/models/manageiq/providers/kubernetes/container_manager/inventory_collections.rb]: Can you please make this an issue? an item is also good

Comment (cben) [inline app/models/manageiq/providers/kubernetes/container_manager/inventory_collections.rb]: => https://github.com/ManageIQ/manageiq-providers-kubernetes/issues/78.  Removing TODO.

**Comment by simon3z** [comment]
@zakiva can you review?

File: app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb [hunk 2]
```diff
@@ -1229,14 +1246,17 @@ def parse_security_context(security_context)
...
-                                                             pod.metadata.namespace,
-                                                             volume.persistentVolumeClaim.try(:claimName))
+        new_result = {
+          :type => 'ContainerVolume',
+          :name => volume.name,
         }.merge!(parse_volume_source(volume))
+        if volume.persistentVolumeClaim.try(:claimName)
+          new_result[:persistent_volume_claim_ref] = {
+            :namespace => pod.metadata.namespace,
+            :name      => volume.persistentVolumeClaim.try(:claimName),
```
Comment (zakiva) [inline app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb]: nit, but I think you can omit the `try`

Comment (cben) [inline app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb]: done

**Comment by moolitayer** [comment]
@cben I thought the test label is for patches that are test but I'm not sure. wdyt?
(If so maybe we should name it test_only)

**Pull request merged**